src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -75,17 +90,69 @@ public void hasPerson_personWithSameIdentityFieldsInAddressBook_returnsTrue() {
         assertTrue(addressBook.hasPerson(editedAlice));
     }
 
+    @Test
+    public void removePerson_personToRemove_personRemoved() {
+        addressBook.addPerson(FIONA);
+        assertTrue(addressBook.hasPerson(FIONA));
+        addressBook.removePerson(FIONA);
+        assertFalse(addressBook.hasPerson(FIONA));
+    }
+
+    @Test
+    public void setPerson_updatedPerson_personUpdated() {
```

Comment:
same goes for here! just a suggestion maybe could change to setPerson_updatePerson_success()